pcdsdevices/epics/pim.py

```diff
@@ -246,7 +246,7 @@ def move_out(self, wait=False, **kwargs):
 
     #Conform to lightpath interface
     remove = move_out
-
+    insert = move_in
```

Comment:
This construct is dangerous if we ever want to use inheritance. If we override `move_in` or `move_out` in a subclass, `insert` and `remove` will still point to exactly these methods, not the new overrides. I used to put this in a lot but I didn't realize the trouble it could cause, so I favor the more wordy `def insert(self, *args, **kwargs): return self.move_in(*args, **kwargs)` pattern.

Reply:
Interesting. I'll make the swap.

Reply:
Just realize that exactly this would have happened in `epics/valve.py` . `GateValve` inherits from `Stopper` which had the shorthand re-aliasing 😮